src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.stories.tsx

```diff
@@ -1,108 +1,21 @@
 /** @jsx h */
```

Comment:
saved this file from chromatic and decided to just link to expansion panel. lonely it doesn't make much sense.